### DISCUSSION TITLE
수평적 조직문화의 매운 맛 해설(페북 펌)

### DISCUSSION THREAD

**Discussion by TaeHyoungKwon** [ThinkAboutSoftware/AcademicConference#182] [General]
페이스북 보다가 괜찮은 글이 하나 있어서 공유드립니다

아래 글의 모든 내용에 공감하진 않습니다(특히 수평적 조직 문화에 대한 부정적으로 말하는 단락은.. 전혀.. 공감이 안되더라구요 아무래도 예시를 일반적인 직급이 있는 기업으로 든 것 같네요)
그럼에도 불구하고, 맨 아래에 내용(볼드 처리한 것)은 저의 개인적으로 매우 동의하는 내용입니다.

단순하게 이 회사의 연봉과 복지혜택의 조건을 보는 것이 아닌, 내가 정말 어떤 것을 하고 싶은지 내가 가려는 회사가 내 비전을 펼칠 수 있는 적합한 회사인지를 판단하는게 중요하다고 생각합니다

더하여서, 실제로 많은 회사들에서 더 좋은 인재를 뽑기 위해서  회사의 가치관, 미션, 컬쳐 덱, 인재상을 강조하고 있고, 앞으로는 더욱더 강조하는 쪽으로 바뀔것이라 생각합니다
한번 읽어 보시면 좋을 것 같네여~

------------

# 수평적 조직문화의 매운 맛 해설

오늘은 약간 흑화된 모드로 수평적 조직문화가 왜 되기 어려운지, 어떻게 해야 하는 지 한번 설명 드리겠습니다.
애플이나 테슬라 같은 회사에 가면 다들 하하호호하면서 널널하게 일할 것 같나요?

“수평적 조직문화“를 원한다는 것은 현상이지 그 이면에 있는 욕구는 크게 간섭 받고 싶지 않다, 존중과 인정을 받고 싶다, 직급과 무관하게 성과로만 평가 받고 싶다, 함께 몰입해서 일하고 싶다. 이겁니다. 그래서 리더들은 이면의 욕구를 잘 읽어야 합니다.

이걸 조금 더 해석하자면 “자유롭게 일하고, 내 실력을 공정하게 인정 받고, 더 많이 받아가고 싶다” 이겁니다. 그러면 여기서부터 과제가 시작됩니다.
자유롭게 일하기는 권한과 책임이 부여될 때 가능합니다. 그러려면 리더는 책임뿐 아니라 권한도 함께 주어야 하고, 실무자는 권한만 갖는게 아니라 책임도 질 줄 알아야 하는데 보통 여기서 충돌이 많이 납니다. 리더는 권한도 안주면서 책임만 지게 하는 것을 권한 위임이라 여기고, 실무자는 권한은 달라고 하면서 책임은 위에서 지게 하는 현상이죠. 근데 실제로 20대 사원, 대리급이 이런 권한과 책임을 질 수 있느냐.. 자율성과 독립성을 갖춘 업무라면 가능하지만 보통 대기업의 경우 사원급이라고 하면 과장급 직원의 업무를 보조하는 역할이지 독립적으로 성과를 만들 기회는 거의 없습니다. 드라마보면 쩔쩔대는 선배직원한테 일침 가하는 인턴이나 신입들 모습 나오죠. 현실에서 그런 일은 없습니다. 역량이 부족하다가 아니라 애초에 신입이 하는 일은 상급자들이 더 생산성있게 일을 할 수 있도록 보조하는 역할이기 때문입니다.

내 실력을 인정 받고 싶은 것은 크게 성과평가와 커뮤니케이션 두 가지로 나눌 수 있습니다. 성과 평가의 핵심은 객관성인데 이건 지표를 몇 백개로 만든다 한들 절대 객관적이라고 인정하지 않는다는 것이 제 경험입니다. 오히려 디테일한 KPI들을 쌓아두는 것보다 어떤 결과가 나와도 서로 수긍할 수 있는 신뢰관계를 구축하는 전략이 훨씬 더 낫습니다. KPI는 성과창출의 도구로 사용되어야지 평가의 답안지가 되어선 안됩니다. 그런데 수많은 회사들은 반대로 하죠. KPI 리스트를 반박할 수 없을 정도로 구체적으로 책으로 만들어서 배포하면 모두가 수긍할 것이라 생각합니다. 대부분 그렇게 실패합니다.

커뮤니케이션은 무조건적으로 발전적 관점을 견지하고 이에 따라 행동하면 됩니다. 말은 쉽지만 실천은 매우 어렵습니다. 코칭, 퍼실리테이션, 비폭력대화 배우시면 도움이 많이 됩니다.

더 많이 받아가고 싶다라는 욕구 때문에 이게 항상 잘 안되는데, 책이나 이런데서 미국 실리콘밸리의 선진 사례가 많이 나옵니다. 그래서 구글이나 넷플릭스 같은 회사의 문화를 베끼려고 많이 합니다. 이러한 모방은 어느 정도 비슷하게 흉내낼 수 있습니다. 그런데 그 회사들 만큼 연봉은 못맞춰 줍니다. 영업이익이 그만큼 나오지 않기 때문입니다. 인생사에 모든 것이 그렇지만 ‘비교‘만큼 사람을 불행하게 느끼게 하는게 없죠. 아무리 억대 연봉이고 잘나가는 국내 대기업이라고 해도 그보다 몇배 더 많이 주는 세계 최고 사례랑 비교하면 당연히 피해의식이 생길 수 밖에 없죠. “나는 지금 내가 일하는 것 이상으로 충분히 보상받고 있다“라고 말하는 임직원을 저는 지금까지 단 한 명도 본 적이 없습니다. 억대연봉자들도 말이죠. 즉 이건 돈의 문제가 아닙니다.

자 그럼 이런 현상을 어떻게 해결해야 하느냐.. 짧은 팁을 드리자면 다음과 같습니다.
이런 고민 자체를 안하게 할 직원을 채용하는 것이 가장 중요합니다. HR의 모든 문제는 결국 채용으로 귀결됩니다. 그럼 눈높이는 실리콘밸리에 가있는 직원들을 어떻게 채용할 것인가..

애초에  조건을 채용의 기본 전제로 보고 결정하기 때문에 이런 문제가 일어납니다. 구직자는 연봉이나 복지를 보고 회사를 결정하고, 회사는 스펙이나 직무 적합성만 보고 뽑으면 당연히 그 조건이 안맞으면 채용되도 언제든지 나갈 수 있죠.
`경력의 관점에서 일터는 목적을 중심으로 결정해야 합니다.`
`내 삶에 있어서 ‘직업‘이란 무엇인지, ‘일’을 통해 내 인생의 가치를 어떻게 발전 시킬 수 있을지. 이런걸 고민해서 회사를 정해야 합니다. 예를 들어 나는 환경에 관심이 있다면 그 삶의 목적을 이룰 수 있는 환경 스타트업에서 일할 수 있을 겁니다.` 경제적 불평등에 관심이 있으면 이를 해결해 줄 금융회사에서 일할 수도 있을 겁니다.

`그런데 거의 모든 커리어 교육들이 이런 내 삶의 목적과 방향 같은건 간과하고 지원 전략이나 자소설 쓰는 법만 가르치니 직장인들은 길을 잃게 되죠.` 회사도 마찬가지 입니다. 채용은 직무 적합도와 조직문화 적합도를 보고 판단해야 하는데 후자는 어떻게 판단할 것인가.. 그 문제지와 답안지를 미리 보여주어야 사람들이 올 수 있습니다. 그 답안지가 바로 `회사의 가치관, 미션, 컬쳐 덱, 인재상` 이런 것들입니다. 이런것이 올곧이 세워지지 않은 상태에서 적당히 괜찮은 사람이라고 뽑아야 오래 다닐 수 없을 수 밖에 없지요. 이런거 없이 일단 대충 회사를 고르니 30대, 40대에도 진로를 어떻게 해야 하느냐는 커리어 컨설팅 시장이 활황입니다. 정신 차리고 보니 길을 잃은 거죠.
이렇게 `회사의 방향과 임직원의 비전이 align 되었을 때 가장 강력한 engagement와 motivation이 일어납니다.` 이건 동기심리학에서 수십년전에 이미 끝난 이야기 입니다.

그렇게 모두가 일치된 목표에 정렬되어 몰입하는 일터가 되면 나머지 부수적인 문제들은 거의 다 해결됩니다.

**Comment by wooyaggo86** [comment]
좋은 글 감사합니다! 옳고 그름, 맞고 틀림을 떠나서 다양한 생각을 해볼 수 있게 하는 글이 좋은 글이라 생각합니다.
저도 이 글도 완전히 공감하지는 않지만 괜찮은 글이라 생각해요👍

**Reply by wooyaggo86** [reply]
글의 목적성을 잘 모르겠지만 아래 두가지라고 간주하고 읽어봤습니다.
- 수평적 조직문화가 왜 되기 어려운지
- 수평적 조직문화가 하기위해 어떻게 해야 하는지

이를 위한 세 가지 요소(욕구)를 다뤄주셨는데요.
- 자유롭게 일한다
    - 권한과 책임이 부여 될 때 가능
        - 한계 - 자율성과 독립성을 갖춘 업무라면 가능하지만,
주니어가 이런 기회를 얻기는 현실적으로는 어렵다.
        - C. 저는 주니어 분들도 자율성과 독립성을 갖춘 업무를 줄 수 있는 환경을 만들어야 한다고 생각하는데, 조직에 따라 다를 수 있겠다란 생각이 드네요.
- 실력을 공정하기 인정 받는다
    - 성가평과
        - 디테일한 KPI를 쌓아두는 것
        - 신뢰관계 구축하는 전략을 추천.
        - C. 저는 이게 둘을 접근법을 적절하게 섞는게 중요하다고 생각합니다. 상호보완적인 관계라고 생각해요.
            - KPI는 내가 아웃풋을 얼마나 냈느냐와 과정을 hightlight하는데 쓰일 수 있습니다. 또 이를 통해, 제가 다른 사람들에게 이사람은 이 증거(KPI등)를 통해서 'A를 미뤄 봤을때, B는 어떤일을 잘하는 것 같아'라고 신뢰를 쌓을 수 있겠죠.
    - 커뮤니케이션
        - `무조건적으로 발전적 관점을 견지하고 이에 따라 행동하면 됩니다.`
            - C. 정말 중요한 부분이라 생각하는데, 답이 없기도하고...
                - 개인적으로는 `Lead with influence instead of authority`가 소통할 때 키워드 아닐까 싶습니다.
- 많은 보상을 받고 싶다
    - 채용으로 해결
        - `경력의 관점에서 일터는 목적을 중심으로 결정해야 합니다.
내 삶에 있어서 ‘직업‘이란 무엇인지, ‘일’을 통해 내 인생의 가치를 어떻게 발전 시킬 수 있을지. 이런걸 고민해서 회사를 정해야 합니다. 예를 들어 나는 환경에 관심이 있다면 그 삶의 목적을 이룰 수 있는 환경 스타트업에서 일할 수 있을 겁니다.`
        - `회사의 방향과 임직원의 비전이 align 되었을 때 가장 강력한 engagement와 motivation이 일어납니다`
        - Q. 이걸 어떻게 채용하는 입장에서 실천할 수 있을까요? 비전, 미션으로 충분할까요?
        - C. 가치에 대해 강조를 하고 많이 공감하기도 합니다만, 가치라는게 한마디로 정의될 수가 없어서 어려운 것 같습니다. 돈, 명예, 경험. 딱 한가지만 뽑기 어렵죠.

**Reply by jongfeel** [reply]
아니 여기서 review가 진행되다니요!? C만 있었다면 그냥 으흠... 하고 지나갈 수 있었을텐데 Q가 있네요?
즐거운 마음으로 답글 달아 봅니다.ㅎㅎㅎㅎ

Q. 이걸 어떻게 채용하는 입장에서 실천할 수 있을까요? 비전, 미션으로 충분할까요?

> 저는 좀 비관적인 생각인데요. 일단 아니라고 봅니다. 회사가 채용하려는 인재에게 비전과 미션 동기부여 보상 등등을 해줄 수 있다고 하더라도 개개인의 만족도를 일정 수준으로 끌어 올리는 것 까지(즉, 상향 평준화)는 가능한데 그 이상은 힘들다고 보거든요.
> 오히려 저는 개인이 회사가 제시한 어떤 기준에 부합한다면, 개인이 이루어 나가야 할 목표는 개인이 설정해서 달성하고 성장하는게 맞는 방향이라고 생각합니다. 그게 돈이나 명예라면 회사가 어느 정도 챙겨줄 수 있겠지만 그것도 어떤 개인한테는 기대에 못미치는 목표일 수도 있거든요.

C. 비유가 맞을지는 모르겠지만 저는 화려한 그래픽과 장엄한 스토리 그리고 평점이 높은 게임에 대해 어쨌든 모두를 대상으로 했을 때를 생각해 본다면 만족도 상향치를 끌어 올려 기대치 만큼 충족해 줄 수 있는 것이라고 봤을 때, TRPG라고 대화로 상황 묘사를 하고 적절한 시스템만 적용시켜서 자유도를 높인 역할 게임에서 더 높은 상상력과 자신의 기대치에 맞는 롤플레잉을 하고 그 만족을 얻을 수 있다고 보면, **개인의 만족은 개인이 설정한 기준**에 따라 다르다는 걸 회사가 받아들여야 하지 않을까요? (Stranger things - 기묘한 이야기에서 demogorgon이 생각나네요. 거기 등장하는 캐릭터들도 D&D라는 TRPG라는 게임을 즐겨합니다ㅎㅎ)

**Comment by jongfeel** [comment]
저도 직장인들이 스스로 생각해 볼 문제라고 봅니다.
내 삶의 목적과 방향이 무엇인가? 를 잘 정해야 하는데 현실을 살아가는데 있어서 그런 건 꿈과 이상이라고 치부하는 것 같습니다.

저 같은 경우도 그런 삶의 목적과 방향을 일찍 정하지 못했습니다.
개발 좀 잘 하고, 게임 하고, 연봉 높은 회사 가서 그럭 저럭 일하고...
그런 삶을 30대 중반까지 살았었으니까요.

지금은 다릅니다.
회사가 만약 제 삶의 목적을 상실하게 하고 방향에 방해를 한다?
그러면 연봉 버리고 미련 없이 나올 수 있을 자신 있습니다.
